README.md

```diff
@@ -33,7 +33,7 @@ cd <built web site directory>
 ../check-links.sh
 ```
 
-The built web site directory will be `production.ecosystemlandscape.linaro.org`.
+The built web site directory will be `production.ecosystemdashboard.linaro.org`.
```

Comment:
I am not sure whether this link is still valid since it gives 404 to me but it might be reachable only to Linaro employees who maintain this repo.